app/src/main/java/com/internship/firstweekapp/arch/ext/BindingExt.kt

```diff
@@ -15,8 +15,20 @@ import com.bumptech.glide.load.engine.GlideException
 import com.bumptech.glide.load.resource.gif.GifDrawable
 import com.bumptech.glide.request.RequestListener
 import com.bumptech.glide.request.RequestOptions
+import com.internship.firstweekapp.ui.game.ZerosCrossesGameBoardView
 import com.internship.firstweekapp.utils.AppUtils
 
+@BindingAdapter("android:userFigureIsCross")
```

Comment:
move this declaration into package with view definition - arch package using for more common binding adapters like circleimage, colorselectors etc